docs/deploying/BBSSignatures.md

```diff
@@ -0,0 +1,27 @@
+# BBS+ Signatures Support
+
+ACA-Py has supported BBS+ Signatures for some time. However, the dependency that is used (`bbs`) does not support ARM architecture, and its inclusion in the default ACA-Py artifacts mean that developers using ARM-based hardware (such as Apple M1 Macs or later) cannot run ACA-Py "out-of-the-box". We feel that providing a better developer experience by supporting ARM architecture is more important than BBS+ Signature support at this time. As such, we have removed the BBS+ dependency from the base ACA-Py artifacts, and made it an add-on, that those using ACA-Py with BBS+ must take extra steps to build their own artifacts. This file describes how to do those extra steps.
+
+Regarding future support for BBS+ Signatures in ACA-Py, there is currently a lot of work going on in developing implementations and BBS+-based Verifiable Credential standards. However, at the time of this release, there is not an obvious approach to an implementation to use in ACA-Py that includes ARM support. As a result, we will hold off on updating the BBS+ Signatures support in ACA-Py until the standards and path forward clarify. In the meantime, maintainers of ACA-Py plan to continue to do all we can to push for newer and better ZKP-based Verifiable Credential standards.
```

Comment:
On this one — change the first comma, to a period (.) and make “There is currently…” a new sentence.